src/server.js

```diff
@@ -17,15 +18,24 @@ dotenv.config();
 const app = express();
 app.response.sendStatus = formattedSendStatus;
 
+const rateLimit = rateLimiter({
+  windowMs: process.env.RATE_WINDOW * 60 * 1000,
```

Comment:
Adicionar comentário sobre a transformação de minutos em milissegundos:
```suggestion
  windowMs: process.env.RATE_WINDOW * 60 * 1000, // x minutes * 60 secs * 1000 ms = time in ms
```